packages/vuetify/src/components/VSelect/VSelect.tsx

```diff
@@ -97,6 +97,11 @@ export const makeVSelectProps = propsFactory({
     role: 'combobox',
   }), ['validationValue', 'dirty', 'appendInnerIcon']),
   ...makeTransitionProps({ transition: { component: VDialogTransition as Component } }),
+  direction: {
```

Comment:
VSelect uses VTextField which uses VInput which already has this prop.

Reply:
Thanks, missed that. Removed